collect_app/src/main/java/org/odk/collect/android/formentry/audit/IdentityPromptViewModel.java

```diff
@@ -59,4 +63,8 @@ private void updateRequiresIdentity() {
     private static boolean userIsValid(String user) {
         return user != null && !user.isEmpty() && !isBlank(user);
     }
+
+    public String getFormName() {
```

Comment:
Super small thing but I'd recommend using `title` rather than `name`. I think `title` is very clearly from the `title` tag but `name` feels more ambiguous to me.

Reply:
Seems fair! Will change